packages/core/status/core-status-server-internal/src/routes/status_preboot.ts

```diff
@@ -17,6 +17,7 @@ export const registerPrebootStatusRoute = ({ router }: { router: IRouter }) => {
       options: {
         authRequired: false,
         tags: ['api'],
+        access: 'public',
```

Comment:
A comment with why the API needs to be public will help us set standards for declaring apis as internal or external. It also helps us communicate why specific API's are public and stable.